orphadata/.htaccess

```diff
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^/fairdatapoint?(.*) http://95.142.173.26:8090/$1 [R=307,L]
```

Comment:
- That first `/` isn't needed.
- The `?` is going to make `t` optional, is that the intent?

Reply:
Hi @davidlehn thanks for the quick response. Thanks for finding my bugs. I wanted something like this `^fairdatapoint/?(.*)`. I will push my changes now